scripts/release-github.js

```diff
@@ -46,7 +46,7 @@ function createRelease (token) {
   /**
    * To match the package version with tags
    */
-  const tagVersion = 'v' + version
+  const tagVersion = `${name}@${version}`
```

Comment:
@jahtalab We are relying on lerna tag creation here. Check details here https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-core#lernapackage